index.ts

```diff
@@ -1,20 +1,32 @@
-import express, {Express} from "express"
+import express, {Express, NextFunction, Request, Response} from "express"
 import dotenv from "dotenv"
 import user_router from "./routers/user"
 import message_router from "./routers/message"
+import bodyParser from "body-parser"
 
 dotenv.config()
 
 const app: Express = express()
 const port = process.env.PORT
 
 app.get('/', (req, res) => {
-    res.send("Welcome to termigram server");
+    res.send("Welcome to termigram server")
 })
 
 app.listen(port, () => {
-    console.log(`[server]: Server is running at http://localhost:${port}`);
+    console.log(`[server]: Server is running at http://localhost:${port}`)
 })
 
+app.use(bodyParser.json())
```

Comment:
I'll need to change way of handling JSON parse exceptions, but that'll do for now I suppose